R/mojquarter.R

```diff
@@ -64,7 +64,7 @@ mojquarter <- function(date, type = "c") {
 
   else if(type == "f") {
 
-    date <- lubridate::ymd(date) - months(3)
+    date <- lubridate::ceiling_date(lubridate::ymd(date), "months") - months(3) - lubridate::days(1)
```

Comment:
This fails when using a date which is not at the end of a month. lubridate::ymd(date) %m-% months(3) works and is probably clearer to use.

Reply:
I've tested the function on the last day of different months and it's worked ok. The function stops working if you try dates like 2020-02-30 or 2021-09-31. Happy to discuss about this.

Reply:
@markrangan cheers for that, makes sense to me...if you fancy writing a decent error message for the function when someone gives a wrong date like that, please feel free!